packages/curl.rb

```diff
@@ -3,108 +3,237 @@
 class Curl < Package
   description 'Command line tool and library for transferring data with URLs.'
   homepage 'https://curl.se/'
-  @_ver = '7.77.0'
+  @_ver = '7.77.0-git'
```

Comment:
Should we remove the curl update from #5837?

Reply:
It's literally the same package file in both PRs. And they're static. So no change is needed.

Reply:
I guess my question is why do we need to update the packages in both PRs?

Reply:
This is only in case you want more testing of the other larger PR. We don't want to leave curl partially broken while that is happening.